Doc/reference/compound_stmts.rst

```diff
@@ -680,8 +680,13 @@ can be used to create instance variables with different implementation details.
 
 .. seealso::
 
-   :pep:`3115` - Metaclasses in Python 3
+   :pep:`3115` - Metaclasses in Python 3000
+      The proposal that introduced the current syntax for declaring metaclasses,
```

Comment:
Perhaps:

The proposal that changed the declaration of metaclasses to the current syntax, and the semantics...